airbyte-integrations/connectors/source-opsgenie/unit_tests/test_source.py

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import unittest
+from unittest.mock import MagicMock
+
+import responses
+from source_opsgenie.source import SourceOpsgenie
+
+
+class SourceOpsgenieTest(unittest.TestCase):
+    def test_stream_count(self):
+        source = SourceOpsgenie()
+        config_mock = MagicMock()
+        streams = source.streams(config_mock)
+        expected_streams_number = 9
```

Comment:
Depending on the number of streams, if we drop `Alerts` this needs to be 8

Reply:
Reintroduced alerts testing. No need to change.